test/integration_test.js

```diff
@@ -719,6 +719,57 @@ describe("Integration tests", function() {
 
       });
 
+      describe("Outgoing conflicting deletion", () => {
+        let id, conflicts;
+
+        beforeEach(() => {
+          return tasks.create({title: "initial"})
+            .then(({data}) => {
+              id = data.id;
+              return tasks.sync();
+            })
+            .then(() => {
+              return tasks.delete(id);
+            })
+            .then(() => {
+              return fetch(`${TEST_KINTO_SERVER}/buckets/default/collections/tasks/records/${id}`, {
+                method: "PUT",
+                headers: {
+                  "Accept":        "application/json",
+                  "Content-Type":  "application/json",
+                  "Authorization": "Basic " + btoa("user:pass"),
+                },
+                body: JSON.stringify({data: {title: "server-updated", done: true}})
+              });
```

Comment:
Haha, good point! Addressed in https://github.com/Kinto/kinto.js/pull/522/commits/7053627f17a5cb2d74f325e353462bcda0b84fd2